codesearch/schema/schema.go

```diff
@@ -32,49 +20,160 @@ const (
 	OwnerField    = "owner"
 	RepoField     = "repo"
 	SHAField      = "sha"
+
+	DefaultSchema = NewDocumentSchema([]types.FieldSchema{
```

Comment:
I probably would hide DefaultSchema behind a function like `GetDefaultSchema()` just because it's a little easier to do stuff to it / ensure callers don't do nasty things, but I don't think it really matters here

Reply:
I waffled on this also, thanks for tipping the scales - changed it to be behind a function